### PR TITLE
Remove GPy and references, update author list

### PR DESCRIPTION
**Pull request by mosesyhc**

### CODE REVIEW

**Comment by wildsm** [comment]
@mosesyhc - I would bring this one in first, is that good?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/PyCQA/flake8
+    rev: '6.1.0'
+    hooks:
+    -   id: flake8
+exclude: '.git|examples/.*|tests/.*'
```
Comment (wildsm) [inline .pre-commit-config.yaml:8]: Is there a reason to exclude `examples/` and `tests/`?

**Comment by mosesyhc** [comment]
@wildsm yes this will have probably the least impact and the necessary updates

File: examples/Example5/.DS_Store
Comment (wildsm) [inline examples/Example5/.DS_Store:1]: @mosesyhc - this file should be deleted

File: .pre-commit-config.yaml (hunk 1 above)
Comment (mosesyhc) [reply .pre-commit-config.yaml:8]: There wasn't, it was an earlier test to make sure CI works. I will add them back in and check flake8 again.

**Review by mosesyhc — COMMENTED** [review]
Meaning list/bibtex in readme should be reordered too. re 4a706f5

**Review by wildsm — APPROVED** [review]
Looks good, thanks for flake8ing the examples.

Once merged, we should remove all of the .DS_Stores under examples/ subdirectories

**Pull request merged**